packages/embark/src/lib/modules/ethereum-blockchain-client/index.js

```diff
@@ -30,7 +33,7 @@ class EthereumBlockchainClient {
     console.dir("== ethereum contract deployer")
     let contractObj = new web3.eth.Contract(contract.abiDefinition, contract.address);
     // let deployObject = this.blockchain.deployContractObject(contractObject, {arguments: contractParams, data: dataCode});
-    let contractObject = contractObj.deploy({ arguments: (contract.args || []), data: ("0x" + contract.code) });
+      let contractObject = contractObj.deploy({ arguments: (contract.args || []), data: ("0x" + contract.code) });
```

Comment:
Random, but this space seems superfluous.

Reply:
Fixed, thanks!